doc/content/modules/user-manual/pages/release-notes/2025.12.0.adoc

```diff
@@ -187,6 +187,18 @@ metadata def MD2 :> SemanticMetadata {
 }
 ```
 
+- The textual import now forces all the _memberFeature_ of _EndFeatureMemberships_ to have _isEnd = true_ like stated in the SysML Specification "8.3.3.3.3 EndFeatureMembership".
+For example, it means that the two features used as connector ends in the following models now have _isEnd = true_.
+
+```
+package pa1 {
+    part pa1;
+    part pa2;
+
+    connect pa1 to pa2; // Here
```

Comment:
Could you be a little more explicit than "// Here" please?

Reply:
Done.